README.md

```diff
@@ -89,7 +96,18 @@ will output
     Assignee: asticode
     Message: This variable should be dropped
     File: mypackage/main.go:6
- 
+
+## Filter by multiple assignees
```

Comment:
I think you can merge this section with the previous one by keeping the title `Filter by multiple assignees` and updating the example command to `go-astitodo -a asticode,username <paths to files or dirs>`